src/puppetlabs/trapperkeeper/services/status/status_core.clj

```diff
@@ -87,11 +102,11 @@
 ;;; Public
 
 (schema/defn ^:always-validate nominal?
-  [status :- ServiceStatus]
+  [status :- ServiceStatus] :- schema/Bool
   (= (:state status) :running))
 
 (schema/defn ^:always-validate all-nominal?
-  [statuses :- ServicesStatus]
+  [statuses :- ServicesStatus] :- schema/Bool
```

Comment:
Schema nnotations for the return value of a function go after the function's name, not its argument vector.